service/src/main/java/bio/terra/cbas/dependencies/bard/BardService.java

```diff
@@ -31,20 +34,36 @@ private DefaultApi getDefaultApi(BearerToken userToken) {
     return bardClient.defaultApi(client);
   }
 
-  public void logRunSetEvent(RunSetRequest request, BearerToken userToken) {
+  public void logRunSetEvent(
+      RunSetRequest request,
+      MethodVersion methodVersion,
+      GithubMethodDetails githubMethodDetails,
+      List<String> workflowIds,
+      BearerToken userToken) {
     String eventName = "workflow-submission";
     HashMap<String, String> properties = new HashMap<>();
     properties.put("runSetName", request.getRunSetName());
-    properties.put("methodVersionId", request.getMethodVersionId().toString());
-    properties.put("wdsRecords", String.valueOf(request.getWdsRecords().getRecordIds().size()));
+    properties.put("methodName", methodVersion.method().name());
+    properties.put("methodSource", methodVersion.method().methodSource());
+    properties.put("methodVersionName", methodVersion.name());
+    properties.put("methodVersionUrl", methodVersion.url());
+    properties.put("recordCount", String.valueOf(request.getWdsRecords().getRecordIds().size()));
+    properties.put("workflowIds", workflowIds.toString());
+
+    if (Objects.nonNull(githubMethodDetails)) {
+      properties.put("githubOrganization", githubMethodDetails.organization());
+      properties.put("githubRepository", githubMethodDetails.repository());
+      properties.put("githubIsPrivate", githubMethodDetails.isPrivate().toString());
```

Comment:
In case either of these sound useful... the method table has a `branchOrTag` field representing the branch name (or tag name!) that the method is coming from. There is also a method version equivalent of this github details with a commit hash field.